owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.java

```diff
@@ -136,7 +137,7 @@ protected void onAccountSet(boolean stateWasRecovered) {
 
```

Comment:
There's still some `// TODO Enable when "On Device" is recovered ?` code in this class